packages/components/src/entities/SampleNav.tsx

```diff
@@ -0,0 +1,86 @@
+/*
+ * Copyright (c) 2018-2019 LabKey Corporation. All rights reserved. No portion of this work may be reproduced in
+ * any form or by any electronic or mechanical means without written permission from LabKey Corporation.
+ */
+import React, { FC, memo, useEffect, useMemo, useState } from 'react';
+import { List } from 'immutable';
+import { WithRouterProps } from 'react-router';
+
+import { useServerContext } from '../internal/components/base/ServerContext';
+import { ITab } from '../internal/components/navigation/types';
+import { SCHEMAS } from '../internal/schemas';
+import { getQueryDetails } from '../internal/query/api';
+import { AppURL } from '../internal/url/AppURL';
+import { SAMPLES_KEY } from '../internal/app/constants';
+import { naturalSortByProperty } from '../public/sort';
+import {
+    isAssayEnabled,
+    isMediaEnabled,
+    isWorkflowEnabled,
+    userCanReadAssays,
+    userCanReadDataClasses,
+} from '../internal/app/utils';
+
+import { SubNav } from '../internal/components/navigation/SubNav';
+
+import { loadSampleTypes } from './actions';
+
+export const SampleTypeDetailsNav: FC<WithRouterProps> = memo(({ params }) => {
```

Comment:
Isn't this a SampleDetailsNav? (It's for a single sample, not for the sample type.)

Reply:
Correct. I've fixed up the names to be `SampleNav` and `SampleTypeNav`

Reply:
make that `SampleIndexNav` and `SampleTypeIndexNav`